pages/k8s/release-notes.md

```diff
@@ -14,6 +14,57 @@ toc: False
 ---
 
 
+# 1.16
+
+### September 23, 2019 -  [charmed-kubernetes-xxx](https://api.jujucharms.com/charmstore/v5/charmed-kubernetes-xxx/archive/bundle.yaml)
+
+## What's new
+
+- Kata Containers support
+
+Beginning with Charmed Kubernetes 1.16, the [Kata Containers](https://katacontainers.io) runtime can be used
+with containerd to safely run insecure or untrusted pods. When enabled, Kata provides hypervisor isolation
+for pods that request it, while trusted pods can continue to run on a shared kernel via runc.
+For details on using Kata Containers with Charmed Kubernetes, consult the [documentation](/kubernetes/docs/kata).
+
+- AWS IAM support
+
+Amazon AWS IAM authentication and authorisation is now supported via a subordinate charm. See
+[AWS-IAM documentation](/kubernetes/docs/aws-iam) for details on how to use AWS credentials
+to log in to your Charmed Kubernetes cluster.
+
+- SSL passthrough support
+
+A new configuration parameter was added to the kubernetes-worker charm to enable ssl passthrough.
+This allows TLS termination to happen on the workload. Refer to the
+[upstream documentation](https://kubernetes.github.io/ingress-nginx/user-guide/tls/#ssl-passthrough)
+for more information.
+
+- Improved LXD support
+
+- Improved Prometheus/Grafana integration
+
+- Improved OpenStack integration
+
+- Docker Registry with Containerd
+
+The Docker registry charm can now be related directly to the Containerd runtime charm.
+Refer [here][docker-registry] for instructions on how to deploy the charm.
```

Comment:
Can you revert these additions - I have added this part to #255 to avoid complicated conflicts

Reply:
- done